Content.Server/Cargo/Systems/CargoSystem.Orders.cs

```diff
@@ -422,17 +432,18 @@ private void UpdateOrderState(EntityUid consoleUid, EntityUid? station)
                     GetOutstandingOrderCount(orderDatabase, console.Account),
                     orderDatabase.Capacity,
                     GetNetEntity(station.Value),
-                    orderDatabase.Orders[console.Account]
+                    orderDatabase.Orders[console.Account],
+                    GetAvailableProducts((consoleUid, console))
                 ));
             }
         }
 
-        private void ConsolePopup(EntityUid actor, string text)
+        internal void ConsolePopup(EntityUid actor, string text) // Moffstation - Made internal for pirates
         {
             _popup.PopupCursor(text, actor);
         }
 
-        private void PlayDenySound(EntityUid uid, CargoOrderConsoleComponent component)
+        internal void PlayDenySound(EntityUid uid, CargoOrderConsoleComponent component)    // Moffstation - Made internal for pirates
```

Comment:
This reuse is pretty gross.
Some of the system is copy/pasted for pirates, some of it is made more public to be directly called. Inconsistent.